### PR TITLE
jitaccess.ops NS delegation

### PR DESCRIPTION
**Pull request by ericmalen**

### CODE REVIEW

File: dns-records/jitaccess.ops.phac.alpha.canada.ca.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+apiVersion: dns.cnrm.cloud.google.com/v1beta1
+kind: DNSRecordSet
+metadata:
+  name: jitaccess-ops-phac-alpha-canada-ca-ns
+  namespace: alpha-dns
+  labels:
+    ownerName: "Eric Malenfant"
+    ownerContact: "eric.richard-malenfant@phac.aspc.gc.ca"
+    gitHubRepo: "JIT-IaC"
```
Comment (KingBain) [inline dns-records/jitaccess.ops.phac.alpha.canada.ca.yaml:9]: could you change that to a URL, even if it's private. I will ask that @sleepycat confirm that's fine.

File: dns-records/jitaccess.ops.phac.alpha.canada.ca.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  labels:
+    ownerName: "Eric Malenfant"
+    ownerContact: "eric.richard-malenfant@phac.aspc.gc.ca"
+    gitHubRepo: "JIT-IaC"
+spec:
+  name: "jitaccess.ops.phac.alpha.canada.ca."
+  type: "NS"
+  ttl: 300
+  managedZoneRef:
+    external: phac-alpha-canada-ca
```
Comment (KingBain) [inline dns-records/jitaccess.ops.phac.alpha.canada.ca.yaml:15]: I think this needs to be 
```suggestion
external: ops-phac-alpha-canada-ca-zone 
```

**Review by KingBain — CHANGES_REQUESTED** [review]
Let me know what you think of the changes

File: dns-records/jitaccess.ops.phac.alpha.canada.ca.yaml (hunk 2 above)
Comment (ericmalen) [reply dns-records/jitaccess.ops.phac.alpha.canada.ca.yaml]: Done

**Pull request merged**